spec/support/ansible_shared/automation_manager/refresher.rb

```diff
@@ -100,12 +100,12 @@
   def assert_counts
     expect(Provider.count).to                                 eq(1)
     expect(automation_manager).to                             have_attributes(:api_version => "3.0.1")
-    expect(automation_manager.configured_systems.count).to    eq(130)
```

Comment:
> The reported count of hosts is more than the actual hosts returned

🤔 

If you have some spare time 😆 maybe you want to add a `docker run` for setting up a tower and populating it?